app/utils/seek-pagination.js

```diff
@@ -0,0 +1,12 @@
+import macro from 'macro-decorators';
+
+export function pagination() {
+  return macro(function () {
+    const { nextPage } = this;
+    const nextPageParams = new URLSearchParams(nextPage);
+
+    return {
+      nextPage: nextPageParams.get('seek'),
+    };
+  });
+}
```

Comment:
Parses the `next_page` value from the response `meta` field which comes a URL Query Params, and retrieves the value from `seek` (if available) otherwise returns `undefined`.